blebox_uniapi/box_types.py

```diff
@@ -270,15 +272,19 @@ def get_latest_api_level(product_type: str) -> Union[dict, int]:
                 "off": lambda x=None: ("GET", "/s/0", None),
             },
             "switches": [
-                ["0.relay", {"state": lambda x: f"relays/[relay={x}]/state"}, "relay"]
+                [
+                    "0.relay",
+                    {"state": lambda x: f"relays[?relay==`{x}`]|[0]|state"},
+                    "relay",
+                ]
             ],
             "sensors": [
                 [
                     "switchBox.energy",
                     {
-                        "energy": "powerMeasuring/powerConsumption/[0]/value",
-                        "periodS": "powerMeasuring/powerConsumption/[0]/periodS",
-                        "measurment_enabled": "powerMeasuring/enabled",
+                        "energy": "powerMeasuring.powerConsumption|[0]|value ",
```

Comment:
trailing empty space in this selector (just after `value`). This is also in few other places.